releases/release-1.34/exceptions.yaml

```diff
@@ -0,0 +1,25 @@
+# Exception requests in v1.34
+# Google Group: https://groups.google.com/g/kubernetes-sig-release
+# Release Team Lead: Vyom Yadav ([@Vyom-Yadav](https://github.com/Vyom-Yadav) 
+
+# Enhancements Freeze Exceptions requested in v1.34
+
+enhancementFreeze:
+
+- name: "In-Place Update of Pod Resources"
+  issue: 1287
+  date_requested: 2025-06-20
+  date_reviewed: 2025-06-23
```

Comment:
For both, is this the date of final approval/rejection or the time discussion was started as the thread was opened on 21st.

Reply:
Yeah, it's fine.